packs/travis_ci/actions/get_git_token.py

```diff
@@ -0,0 +1,22 @@
+from lib.action import TravisCI
+import yaml
+import requests
+
+
+class GetGitToken(TravisCI):
+    def run(self):
+        username = self.config['username']
+        password = self.config['password']
+        url = 'https://api.github.com/authorizations'
+        host = 'api.github.com'
+        ContentType = 'application/json'
+        data = '{"scopes": ["read:org", "user:email",\
```

Comment:
Please use a dict and `json.dumps` to create this JSON payload. Creating JSON using a string is not robust and also unsafe in many scenarios.